liquibase-core/src/main/java/liquibase/sqlgenerator/core/util/MSSQLUtil.java

```diff
@@ -0,0 +1,67 @@
+package liquibase.sqlgenerator.core.util;
+
+import liquibase.sql.Sql;
+import liquibase.sql.UnparsedSql;
+import liquibase.structure.core.Relation;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * MSSQL Util for generator to handle SET methods for now.
+ * Additional utility methods can be added here
+ */
+public class MSSQLUtil {
+    public static String IS_SET_REGEX = "(?i)SET\\s+(?i)(ANSI_NULLS|QUOTED_IDENTIFIER)\\s(?i)(ON|OFF)(\\s*\\t*\\n*);?";
```

Comment:
@bbrown-at-datical , what do you think about this regexp, will it handle all possible variations of `SET ANSI_NULLS/QUOTED_IDENTIFIER ... `?